swift/tools/swiftbuild.py

```diff
@@ -147,4 +138,17 @@ def build_package(test, run, package, swiftc_options):
     if args.parseable:
         swiftc_options.append("-parseable-output")
 
+    # If no package list provided, use all directories in the
+    # current working directory.
+
```

Comment:
(nit) remove the blank line to keep the comment tightly attached to the arg processing block.

Reply:
Done. Did the same a couple other places, too.